packages/shared/src/components/notifications/NotificationItemAvatar.tsx

```diff
@@ -7,6 +7,9 @@ import SourceButton from '../cards/common/SourceButton';
 import { ProfileTooltip } from '../profile/ProfileTooltip';
 import { ProfileImageLink } from '../profile/ProfileImageLink';
 import { ProfileImageSize } from '../ProfilePicture';
+import { MedalBadgeIcon } from '../icons/MedalBadge';
```

Comment:
I think for icons we said we would keep using barrel imports as the only exception for now :)

Reply:
Changed to barrel